packages/plugins/quorum/src/index.js

```diff
@@ -0,0 +1,81 @@
+import { __ } from 'embark-i18n';
+const {testRpcWithEndpoint, testWsEndpoint} = require('embark-utils');
```

Comment:
Can it be changed to:

```suggestion
import {testRpcWithEndpoint, testWsEndpoint} from 'embark-utils';
```

Reply:
Good catch, updated!